### PR TITLE
Remove deprecated email.yml setting file

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: _includes/manuals/1.16/3.5.2_configuration_options.md [hunk 1]
```diff
@@ -83,68 +83,9 @@ Some plugins may add their own loggers. See the configuration files in /etc/fore
 
 #### The 'email.yaml' file
 
-This settings file can be found at `/etc/foreman/email.yaml`, or `config/email.yaml` on a source installation.
+**This file has been deprecated for configuring email, SMTP and sendmail options. Please remove it and configure via the web UI under *Administer > Settings > Email* instead. On Foreman 1.16 or later, this file will be completely useless.**
```
Comment (ekohl) [inline _includes/manuals/1.16/3.5.2_configuration_options.md]: I think this should be in the Upgrade guide and release notes. The reference manual shouldn't document the outdated section.

**Review by ekohl — CHANGES_REQUESTED** [review]

**Comment by tbrisker** [comment]
i started digging a bit deeper and found other references to email.yaml that led me to discover we still support file based email config in the installer.... opened https://projects.theforeman.org/issues/24399 to track that.
I'm out for the weekend, i'll continue with removing this from the manual next week.

**Comment by ekohl** [comment]
Thanks! https://github.com/theforeman/puppet-foreman/pull/656 should take care of that. Now we're getting back in the regular releasing and cleaning up.

**Comment by tbrisker** [comment]
I hope I got all of them now.

**Review by ekohl — APPROVED** [review]

**Pull request merged**